applications.json

```diff
@@ -1902,9 +1902,50 @@
       },
       {
         "kind": "Class",
-        "id": "WindowsForms10.Window.8.app.0.13fa1bf_r22_ad1",
+        "id": "WindowsForms10.Window",
```

Comment:
This seems to be a bit overzealous. Remote Desktop Manager also has windows with this prefix, which all of a sudden started getting ignored for me when I updated the application specific configuration rules.

Reply:
I think we should make this a composite rule scoped to the Visual Studio executable

Reply:
@CtByte could you submit an update for this?

Reply:
@LGUG2Z @gazpachoking Will [this](https://github.com/LGUG2Z/komorebi-application-specific-configuration/pull/120) do? 😅